Maestro/subscriptions.json

```diff
@@ -410,6 +410,34 @@
         }
       }
     },
+    // Update dependencies in core-setup dev/defaultintf
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/dev/defaultintf/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/dev/defaultintf/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/dev/defaultintf/Latest.txt"
+      ],
+      "action": "core-setup-general",
+      "delay": "00:10:00",
+      "actionArguments": {
+        "vsoSourceBranch": "dev/defaultintf",
+        "vsoBuildParameters": {
+          "ScriptFileName": "build.cmd",
+          "Arguments": [
+            "--",
+            "/t:UpdateDependenciesAndSubmitPullRequest",
+            "/p:GitHubUser=dotnet-bot",
+            "/p:GitHubEmail=dotnet-bot@microsoft.com",
+            "/p:GitHubAuthToken=`$(`$Secrets['DotNetBotGitHubPassword'])",
+            "/p:ProjectRepoOwner=dotnet",
+            "/p:ProjectRepoName=core-setup",
+            "/p:ProjectRepoBranch=dev/defaultintf",
+            "/p:NotifyGitHubUsers=dotnet/core-setup-contrib",
```

Comment:
Might be better to not notify this group for now, to avoid noise while this branch gets off the ground. (Just remove this line.)